### ISSUE TITLE
Library Mismatch on Windows

### ISSUE THREAD

**Issue by Kai-Indiemade** [NVIDIA-Omniverse/PhysX#134]
Hi there, I've linked the .lib files to my project using cmakelists, however I git this error:
LNK2038 mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in test.main.obj
File: D:\VisualStudioProjects\test\out\build\x64-debug\PhysXExtensions_static_64.lib(ExtDefaultErrorCallback.obj)

I don't know if this is normal, but the following libraries have _static in the name, the others do not:
PhysXExtensions_static_64.lib
PhysXPvdSDK_static_64.lib
PhysXVehicle_static_64.lib
PhysXCharacterKinematic_static_64.lib

I think that is the problem. I tried setting these to False in the build preset, but there seemed to be no difference:
PX_GENERATE_STATIC_LIBRARIES
NV_USE_STATIC_WINCRT

I am using a custom preset, but it's just a copy-paste of vc16win64.xml just to change it to use vc17 so that I could build the project with Visual Studio 2022.

I've seen older issues where people had the same version, but I was unable to solve the issue with the suggested solutions in those threads.

**Comment by preist-nvidia** [comment]
Yes, this sounds like a CRT config mismatch between PhysX and your project. 

We cannot help much with build issues, but please also see here in our docs at the end: https://nvidia-omniverse.github.io/PhysX/physx/5.1.3/docs/BuildingWithPhysX.html
